Dockerfile-multistage-node

```diff
@@ -36,6 +36,18 @@ COPY --from=node-build /hydroshare /hydroshare
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     locale-gen
 
+RUN pip install psycopg==3.1.*
```

Comment:
Do we need to remove the install commands from here?
https://github.com/hydroshare/hydroshare/pull/5352/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R8

Reply:
no they are intentional there until we get them into the base image